docker/readme

```diff
@@ -0,0 +1,2 @@
+For docker build use: docker build https://github.com/anika-is-here/spring-petclinic.git#{$BUILD_BRANCH} -f ./docker/Dockerfile_app -t petclinic-app
```

Comment:
It's better to put such instructions into Readme.md with appropriate [markdown](https://guides.github.com/features/mastering-markdown/) so that github will automatically recognize it and show in UI.

Reply:
Ready.